src/lib/Components/Bidding/Components/Grid.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react"
+import { Flex } from "../Elements/Flex"
+
+export const Row = props => <Flex flexDirection="row" justifyContent="space-between" {...props} />
+export const Col = props => <Flex flex={1} {...props} />
```

Comment:
Different layouts will have their own row and Column needs, but it seems like something like this should be an Element primitive that accepts props. One-off rows / columns are responsible for a lot of design-drift so at this stage we should discourage it.

Reply:
Agreed, we should use the same `Row` and `Column` wherever possible in order to standardize grid system and our margin/padding values.